index.html

```diff
@@ -52,6 +61,234 @@ <h2 class="sub-title">I am a Softwaer Developer</h2>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+            
+        </ul>
+    </div>
+    <div class="card-last-button">See Project</div>
+
+   </section>
+   <!--about section start-->
+   <section class="card-3">
+    <div class="card-about-title">About me</div>
+    <div class="card-about-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
+    <div class="card-about-button">Get My Resume</div>
+    <div class="about-img"><img src="img/Vector44.png" alt=""></div>
+    <!--skill part start-->
+    <div class="skill">
+       
+        <div class="card-skill">
+            
+            <h3 class="skill-img ab"> <img src="img/skill55.png" alt=""> Language</h3>
+            <ul>
+                <li class="skillname sn">
+                    <a href="#">Codekit</a>
+                </li>
+                <li class="skillname sn">
+                    <a href="#">Github</a>
+                </li>
+                <li class="skillname sn">
+                    <a href="#">Codepen</a>
+                </li>
+                <li class="skillname sn">
+                    <a href="#">Gitlap</a>
+                </li>
+                <li class="skillname sn">
+                    <a href="#">Terminal</a>
+                </li>
+            </ul>
+        </div>
+        <div class="card-skill">
+            
+            <h3 class="skill-img ab"> <img src="img/framwork56.png" alt=""> Framework</h3>
+            <ul>
+                <li class="skillname sn">
+                    <a href="#">Codekit</a>
+                </li>
+                <li class="skillname sn">
+                    <a href="#">Github</a>
+                </li>
+                <li class="skillname sn">
+                    <a href="#">Codepen</a>
+                </li>
+                <li class="skillname sn">
+                    <a href="#">Gitlap</a>
+                </li>
+                <li class="skillname sn">
+                    <a href="#">Terminal</a>
+                </li>
+            </ul>
+        </div>
+        <div class="card-skill">
+            
+            <h3 class="skill-img"> <img src="img/round57.png" alt=""> Skill</h3>
+            <ul>
+                <li class="skillname">
+                    <a href="#">Codekit</a>
+                </li>
+                <li class="skillname">
+                    <a href="#">Github</a>
+                </li>
+                <li class="skillname">
+                    <a href="#">Codepen</a>
+                </li>
+                <li class="skillname">
+                    <a href="#">Gitlap</a>
+                </li>
+                <li class="skillname">
+                    <a href="#">Terminal</a>
+                </li>
+            </ul>
+        </div>
+    </div>
+   </section>
```

Comment:
- [ ] Great job implementing the about section 👌 However, please make sure that you follow the figma template. For more clarity, please visit the above comments 👆
- [ ] Please make sure to add a `background-image` for your about section. Currently, your about section lacks an image
- [ ] Please make sure to style for button properly. I think in about section you forgot to add colors for that. Don't worry you can fix this. Don't forget to add proper button interactions according to the figma template.
- [ ] Please make sure that your `font-weight` and `font-size` is in accordance to figma template

### Your design 👇
![image](https://user-images.githubusercontent.com/106119105/212502567-278efc16-b5cd-47ab-965f-04b5e4668c9c.png)

### Figma design 👇
![image](https://user-images.githubusercontent.com/106119105/212502581-fdca8507-9346-4b2c-b3e6-b90f1dec31b5.png)